### PR TITLE
Update geo groups to be configured similar to low-income thresholds.

### PR DESCRIPTION
**Pull request by datu925**
The implementation in refineCollectedData is a little repetitive, but I didn't see an obvious way to refactor it.

### CODE REVIEW

File: test/data/schemas.test.ts
Comment (datu925) [inline test/data/schemas.test.ts:227]: Just flagging that I moved these into the new test file to keep geogroup-related tests together. The middle test (nonexistent group ID) is implicitly covered by the test that verifies equivalence of config and incentives.

File: scripts/lib/data-refiner.ts
Comment (oyamauchi) [inline scripts/lib/data-refiner.ts:139]: Yeah, seeing it now, I like that the `-group-` thing is no longer load-bearing.

**Review by oyamauchi — APPROVED** [review]
This will conflict with #345 , so could you wait until that's landed and fix up the NV geo groups file before landing this? (Or coordinate with Rachel to do it in the opposite order)

**Pull request merged**
